### PR TITLE
fix : Null object error fixed in track details items

### PR DESCRIPTION
**Pull request by dreadpool2**
Changes: 'To be decided' microlocations were left blank in items, so added a 'Not yet decided' line.
App doesn't crash on clicking certain items in the tracks list. (which it used to)

Screenshots for the change: 
screenshot_20171117-123206

### CODE REVIEW

**Comment by dreadpool2** [comment]
@the-dagger @iamareebjamal Please review

File: android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java [hunk 1]
```diff
@@ -179,9 +179,12 @@ public void onBindViewHolder(final SessionViewHolder holder, final int position)
         holder.sessionTime.setText(String.format("%s - %s",
                 DateConverter.formatDateWithDefault(DateConverter.FORMAT_12H, session.getStartsAt()),
                 DateConverter.formatDateWithDefault(DateConverter.FORMAT_12H, session.getEndsAt())));
+
         if(session.getMicrolocation() != null) {
             String locationName = Utils.checkStringEmpty(session.getMicrolocation().getName());
             holder.sessionLocation.setText(locationName);
+        } else{
+            holder.sessionLocation.setText("Not Decided Yet");
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java]: Please move this to strings.xml.
Thanks 😄

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

**Comment by dreadpool2** [comment]
@iamareebjamal Did the change!

File: android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java [hunk 2]
```diff
@@ -179,9 +179,12 @@ public void onBindViewHolder(final SessionViewHolder holder, final int position)
         holder.sessionTime.setText(String.format("%s - %s",
                 DateConverter.formatDateWithDefault(DateConverter.FORMAT_12H, session.getStartsAt()),
                 DateConverter.formatDateWithDefault(DateConverter.FORMAT_12H, session.getEndsAt())));
+
         if(session.getMicrolocation() != null) {
             String locationName = Utils.checkStringEmpty(session.getMicrolocation().getName());
             holder.sessionLocation.setText(locationName);
+        } else{
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java]: Please add space after `}`

File: android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java [hunk 3]
```diff
@@ -464,13 +464,14 @@ protected void onResume() {
 
     private void loadSession(Session session) {
         this.session = session;
-        this.trackColor = Color.parseColor(session.getTrack().getColor());
-        this.fontColor = Color.parseColor(session.getTrack().getFontColor());
-        this.darkColor = Views.getDarkColor(Color.parseColor(session.getTrack().getColor()));
-
         Track track = session.getTrack();
+
+
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java]: Please remove this extra line

File: android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java (hunk 2 above)
Comment (dreadpool2) [reply android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java]: @iamareebjamal SIr, isn't there already a space given ?

**Comment by dreadpool2** [comment]
@iamareebjamal done the changes

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**